apps/condo/domains/common/utils/addressApi/addressMetaGql.js

```diff
@@ -0,0 +1,190 @@
+const ADDRESS_META_GRAPHQL_TYPES = `
```

Comment:
Please move it inside property domain (not common)

Reply:
It is being used by 'Property' and 'Resident' — two different domains.
What the reason to move it to property domain?

Reply:
Moved